ci-operator/jobs/konflux-ci/e2e-tests/konflux-ci-e2e-tests-main-presubmits.yaml

```diff
@@ -135,7 +135,7 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )load-test-ci-100u-1t,?($|\s.*)
+    trigger: (?m)^/test( | .* )(load-test-ci-100u-1t|remaining-required),?($|\s.*)
```

Comment:
what does this change do?

Reply:
this is a regex that's used for triggering the job by commenting `/test <string-that-matches-regex>` on a PR

Reply:
the change was introduced automatically by prow job config automation (`make jobs`)

Reply:
ack, thanks!